src/include/error.h

```diff
@@ -124,3 +124,17 @@
 #define	WT_ASSERT(session, exp)						\
 	WT_UNUSED(session)
 #endif
+
+/*
+ * __wt_verbose --
+ *	Display a verbose message. Not an inlined function because you can't
+ * inline functions that take variadic arguments.
```

Comment:
I would add a comment here about the requirement for at least one arg so someone a few years from now doesn't start down the path to undo that.  Admittedly I find that requirement a bit awkward and not pretty.